internal/streaming/clickhouse_connector/connector.go

```diff
@@ -0,0 +1,201 @@
+		EventsTableName: getEventsTableName(namespace),
+		EventType:       meter.EventType,
+		MeterViewName:   getMeterViewName(namespace, meter),
+		ValueProperty:   meter.ValueProperty,
+		GroupBy:         meter.GroupBy,
+	})
+	if err != nil {
+		return err
+	}
+	err = c.config.ClickHouse.Exec(ctx, query)
+	if err != nil {
+		return fmt.Errorf("create meter view: %w", err)
+	}
+
+	return nil
+}
+
+func (c *ClickhouseConnector) queryMeterView(ctx context.Context, namespace string, meter *models.Meter, params *streaming.GetValuesParams) ([]*models.MeterValue, error) {
+	values := []*models.MeterValue{}
+
+	groupBy := make([]string, 0, len(meter.GroupBy))
+	for key := range meter.GroupBy {
+		groupBy = append(groupBy, key)
+	}
+
+	query, err := streaming.TemplateQuery(queryMeterViewTemplate, queryMeterViewData{
+		Database:      c.config.Database,
+		MeterViewName: getMeterViewName(namespace, meter),
+		Subject:       params.Subject,
+		From:          params.From,
+		To:            params.To,
+		GroupBy:       groupBy,
+		// TODO: implement window size
+		WindowSize: params.WindowSize,
+	})
+	if err != nil {
+		return values, err
+	}
+	rows, err := c.config.ClickHouse.Query(ctx, query)
+	if err != nil {
+		return values, fmt.Errorf("query meter view query: %w", err)
+	}
+
+	for rows.Next() {
+		value := &models.MeterValue{
+			GroupBy: map[string]string{},
+		}
+		args := []interface{}{&value.WindowStart, &value.WindowEnd, &value.Subject, &value.Value}
+		// TODO: do this next part without interface magic
+		for range groupBy {
+			tmp := ""
+			args = append(args, &tmp)
+		}
+
+		if err := rows.Scan(args...); err != nil {
+			return values, fmt.Errorf("query meter view row scan: %w", err)
+		}
+
+		for i, key := range groupBy {
+			if s, ok := args[i+4].(*string); ok {
+				value.GroupBy[key] = *s
+			}
+		}
+
+		values = append(values, value)
+	}
+	rows.Close()
+	err = rows.Err()
+	if err != nil {
+		return values, fmt.Errorf("query meter rows error: %w", err)
+	}
+
+	return values, nil
+}
+
+func (c *ClickhouseConnector) createSinkConnector(ctx context.Context, namespace string) error {
+	connector := &sink.Connector{
+		Name: "clickhouse",
+		Config: map[string]string{
+			"connector.class":                "com.clickhouse.kafka.connect.ClickHouseSinkConnector",
+			"database":                       "default",
+			"errors.retry.timeout":           "30",
+			"hostname":                       "clickhouse",
+			"port":                           "8123",
+			"ssl":                            "false",
+			"username":                       "default",
+			"password":                       "",
+			"key.converter":                  "org.apache.kafka.connect.storage.StringConverter",
+			"value.converter":                "org.apache.kafka.connect.json.JsonConverter",
+			"value.converter.schemas.enable": "false",
+			"schemas.enable":                 "false",
+			"topics":                         getEventsTableName(namespace),
```

Comment:
TODO: check if wildcard works to match all events topic instead of connector per namespaces

Reply:
it does